caniusepython3/dependencies.py

```diff
@@ -78,7 +78,12 @@ def blocking_dependencies(projects, py3_projects):
     evaluated = set()
     for project in projects:
         log.info('Checking top-level project: {0} ...'.format(project))
-        dist = distlib.locators.locate(project)
+        try:
+            dist = distlib.locators.locate(project)
+        except AttributeError:
+            # This is a work around //bitbucket.org/pypa/distlib/issue/59/
```

Comment:
Please end the comment in a period so readers know the sentence is actually done and not a copy-and-paste error.